Dockerfile

```diff
@@ -1,6 +1,18 @@
 # Dockerfile used to build the VM image which will be downloaded by providers.
 # The file must specify a workdir and at least one volume.
+
+# We're using python slim image in this example to limit the time it takes for the
+# resultant image to be first downloaded by providers, given the fact that our example
+# here is limited to barebones Python installation.
+#
+# In a production app, one might wish to utilize a more full-featured Python image.
```

Comment:
Actually, I think trying to use minimal images in production apps can be considered a good practice (for the sake of the final image size). Therefore, I'd skip this sentence from the comment.